techniques/systemSettings/userManagement/userManagement/8.0/userManagement.ps1.st

```diff
@@ -0,0 +1,108 @@
+function check_usergroup_user_parameters {
+  [CmdletBinding()]
+  param (
+      [parameter(Mandatory=$true)]  [string]$reportId,
+      [parameter(Mandatory=$true)]  [string]$techniqueName,
+      [switch]$auditOnly
+  )
+    
+    $trackingkey = @(
+    &TRACKINGKEY:{directiveId |
+    "&directiveId&" };separator=","& )
+
+    $logins = @(
+    &USERGROUP_USER_LOGIN:{login |
+    "&login&" };separator=","& )
+
+    $policies = @(
+    &USERGROUP_USER_ACTION:{policy |
+    "&policy&" };separator=","& )
+
+    $passwords = @(
+    &USERGROUP_USER_PASSWORD:{password |
+    "&password&" };separator=","& )
+
+    $password_policies = @(
+    &USERGROUP_USER_PASSWORD_POLICY:{password_policy |
+    "&password_policy&" };separator=","& )
+
+    $present        = "add"
+    $absent         = "remove"
+#To REMOVE in the future
```

Comment:
why?

Reply:
it is to keep compatibility with old versions... But now we have the Audit mode to do that feature.
To remove when making a new clean version in dsc AND cf-engine.